src/ipfs.ts

```
@@ -1,60 +1,198 @@
-import { json } from "@helia/json";
-import { createHelia } from "helia";
+import { json, JSON } from "@helia/json";
```

Biome (2.1.2)

[error] Do not shadow the global "JSON" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)